Changelog.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## Next
+
+- Parser:
+    - Fix parsing of ranges in `UnicodeData.txt`.
+    - Generate `lib/Unicode/Internal/Char/UnicodeData/GeneralCategory.hs`.
+- Library:
+    - Support for big-endian architectures.
+    - Add `GeneralCategory` data type and corresponding `generalCategoryAbbr, generalCategory` functions.
+    - Add the following functions: `isAlpha, isAlphaNum, isControl, isMark, isNumber, isPrint, isPunctuation, isSeparator, isSymbol`.
+    - **Breaking change:** `isLetter` is renamed `isAlphabetic` and `isSpace` is renamed `isWhiteSpace`.
```

Comment:
- Added `isAlphabetic` and `isWhiteSpace`.
- Changed the behavior of `isLetter` and `isSpace` to match base's `Data.Char` behavior.

Reply:
Fixed